scrapers/Algolia.py

```diff
@@ -457,65 +382,74 @@ def scraping_json(api_json, url=None):
 SCENE_TITLE = FRAGMENT.get("title")
 SCENE_URL = FRAGMENT.get("url")
 
+#log.debug(HEADERS)
+#log.debug('{}'.format(FRAGMENT))
+
+
 if "validName" in sys.argv and SCENE_URL is None:
-    sys.exit()
+    sys.exit(1)
 
 if SCENE_URL and SCENE_ID is None:
-    debug("[DEBUG] URL Scraping: {}".format(SCENE_URL))
+    log.debug("URL Scraping: {}".format(SCENE_URL))
 else:
-    debug("[DEBUG] Stash ID: {}".format(SCENE_ID))
-    debug("[DEBUG] Stash Title: {}".format(SCENE_TITLE))
+    log.debug("Stash ID: {}".format(SCENE_ID))
+    log.debug("Stash Title: {}".format(SCENE_TITLE))
+
 # Get your sqlite database
-DB_PATH = graphql_configdb()
+stash_config = graphql.configuration()
+DB_PATH = None
+if stash_config:
+    DB_PATH = stash_config["general"]["databasePath"]
 if (CONFIG_PATH and DB_PATH is None):
+    # getting your database from the config.yml
     if (os.path.isfile(CONFIG_PATH)):
         with open(CONFIG_PATH) as f:
             for line in f:
                 if "database: " in line:
                     DB_PATH = line.replace("database: ", "").rstrip('\n')
                     break
-debug("[DEBUG] Database Path: {}".format(DB_PATH))
+log.debug("Database Path: {}".format(DB_PATH))
 if DB_PATH:
     if SCENE_ID:
         # Get data by GraphQL
-        database_dict = graphql_getscene(SCENE_ID)
+        database_dict = graphql.getScene(SCENE_ID)
         if database_dict is None:
             # Get data by SQlite
-            debug("[WARN] Fail to use GraphQL, trying to read database directly.")
+            log.warning("Fail to use GraphQL, trying to read database directly.")
             database_dict = check_db(DB_PATH, SCENE_ID)
         else:
             database_dict = database_dict["file"]
-        debug("[DATABASE] Info: {}".format(database_dict))
+        log.debug("[DATABASE] Info: {}".format(database_dict))
     else:
         database_dict = None
-        debug("[WARN] Can't connect to the database because URL Scraping.")
+        log.warning("Can't use the database because URL Scraping.")
```

Comment:
Not sure about this one here also....
Imho is should be `debug` or `info` level not warning.
Also i think its not can't but wont? Anyway something like this ?
```py
  log.debug("URL scraping... Ignoring database...")
```
is enough